mage_ai/io/bigquery.py

```diff
@@ -37,36 +39,49 @@ def __init__(self, **kwargs) -> None:
         All keyword arguments except for `path_to_credentials` and `credentials_mapping` will be
         passed to the Google BigQuery client, accepting all other configuration settings there.
         """
-        if kwargs.get('verbose') is not None:
-            kwargs.pop('verbose')
-        super().__init__(verbose=kwargs.get('verbose', True))
-
-        credentials = kwargs.get('credentials')
+        if kwargs.get("verbose") is not None:
+            kwargs.pop("verbose")
+        super().__init__(verbose=kwargs.get("verbose", True))
+
+        CLIENT_SCOPES = [
+            "https://www.googleapis.com/auth/userinfo.email",
+            "https://www.googleapis.com/auth/cloud-platform",
+            "https://www.googleapis.com/auth/sqlservice.login",
```

Comment:
Is this scope needed?

Reply:
https://cloud.google.com/bigquery/docs/external-data-drive#python

Looks like we might only need drive & bq. I borrowed these from dbt. I'll test

Reply:
Confirmed this works with the following scopes, which match googles docs

```python
CLIENT_SCOPES = [
            "https://www.googleapis.com/auth/drive",
            "https://www.googleapis.com/auth/bigquery",
        ]
```

Reply:
I would say I'm marginally concerned this could break something if a google drive api isn't enabled